### ISSUE TITLE
pm2 list and status show wrong user

### ISSUE THREAD

**Issue by pepez** [Unitech/pm2#3360]
## What's going wrong?
pm2 list and status show current user instead of user running the application. In the example below the PID 47988 is running as user 501 (pm2-user).
```
[root@server ~]# ps aux | grep -i pm2
501      21566  0.0  0.7 936828 31252 ?        Ssl  Dec18   0:15 PM2 v2.7.2: God Daemon (/data/shop/.pm
root     48490  0.0  0.0 103300   800 pts/0    S+   10:04   0:00 grep -i pm2
[root@server ~]# ps aux | grep -i node
501      47988  0.4  1.4 1258148 59968 ?       Ssl  09:44   0:05 node /data/shop/shop-25/server-bundle.js      
root     48520  0.0  0.0 103296   792 pts/0    S+   10:05   0:00 grep -i node
[root@server ~]# pm2 list
┌───────────────┬────┬──────┬───────┬────────┬─────────┬────────┬─────┬───────────┬──────┬──────────┐
│ App name      │ id │ mode │ pid   │ status │ restart │ uptime │ cpu │ mem       │ user │ watching │
├───────────────┼────┼──────┼───────┼────────┼─────────┼────────┼─────┼───────────┼──────┼──────────┤
│ shop-25       │ 2  │ fork │ 47988 │ online │ 0       │ 20m    │ 0%  │ 58.5 MB   │ root │ disabled │
└───────────────┴────┴──────┴───────┴────────┴─────────┴────────┴─────┴───────────┴──────┴──────────┘
 Use `pm2 show <id|name>` to get more details about an app
[root@server ~]# su - pm2-user
[pm2-user@server ~]$ pm2 list
┌───────────────┬────┬──────┬───────┬────────┬─────────┬────────┬─────┬───────────┬────────────┬──────────┐
│ App name      │ id │ mode │ pid   │ status │ restart │ uptime │ cpu │ mem       │ user       │ watching │
├───────────────┼────┼──────┼───────┼────────┼─────────┼────────┼─────┼───────────┼────────────┼──────────┤
│ shop-25       │ 2  │ fork │ 47988 │ online │ 0       │ 20m    │ 0%  │ 58.5 MB   │ pm2-user   │ disabled │
└───────────────┴────┴──────┴───────┴────────┴─────────┴────────┴─────┴───────────┴────────────┴──────────┘
 Use `pm2 show <id|name>` to get more details about an app

```

```
$ pm2 report
===============================================================================
--- PM2 REPORT (Tue Dec 19 2017 09:59:49 GMT+0200 (EET)) ----------------------
===============================================================================
--- Daemon -------------------------------------------------
pm2d version         : 2.7.2
node version         : 8.9.1
node path            : /usr/bin/pm2
argv                 : /usr/bin/node,/usr/lib/node_modules/pm2/lib/Daemon.js
argv0                : node
user                 : pm2-user
uid                  : 501
gid                  : 501
uptime               : 1028min
===============================================================================
--- CLI ----------------------------------------------------
local pm2            : 2.7.2
node version         : 8.9.1
node path            : /usr/bin/pm2
argv                 : /usr/bin/node,/usr/bin/pm2,report
argv0                : node
user                 : root
uid                  : 0
gid                  : 0
===============================================================================
--- System info --------------------------------------------
arch                 : x64
platform             : linux
type                 : Linux
cpus                 : Intel(R) Xeon(R) CPU E5-2697 v2 @ 2.70GHz
cpus nb              : 1
freemem              : 156311552
totalmem             : 4145758208
home                 : /root
===============================================================================
--- PM2 list -----------------------------------------------
┌───────────────┬────┬──────┬───────┬────────┬─────────┬────────┬─────┬───────────┬──────┬──────────┐
│ App name      │ id │ mode │ pid   │ status │ restart │ uptime │ cpu │ mem       │ user │ watching │
├───────────────┼────┼──────┼───────┼────────┼─────────┼────────┼─────┼───────────┼──────┼──────────┤
│ shop-25       │ 2  │ fork │ 47988 │ online │ 0       │ 14m    │ 0%  │ 76.2 MB   │ root │ disabled │
└───────────────┴────┴──────┴───────┴────────┴─────────┴────────┴─────┴───────────┴──────┴──────────┘
===============================================================================
--- Daemon logs --------------------------------------------
┌───────────────┬────┬──────┬───────┬────────┬─────────┬────────┬─────┬───────────┬──────┬──────────┐
│ App name      │ id │ mode │ pid   │ status │ restart │ uptime │ cpu │ mem       │ user │ watching │
├───────────────┼────┼──────┼───────┼────────┼─────────┼────────┼─────┼───────────┼──────┼──────────┤
│shop-25        │ 2  │ fork │ 47988 │ online │ 0       │ 14m    │ 0%  │ 76.2 MB   │ root │ disabled │
└───────────────┴────┴──────┴───────┴────────┴─────────┴────────┴─────┴───────────┴──────┴──────────┘
 Use `pm2 show <id|name>` to get more details about an app

```

**Comment by Unitech** [comment]
could you try with latest pm2 version (`npm install pm2 -g; pm2 update`)

**Comment by pepez** [comment]
No difference with 2.9.1

**Comment by Unitech** [comment]
could you run `pm2 prettylist` under root and pm2-user and copy/paste this here, thanks

**Comment by Unitech** [comment]
Thanks to @wallet77, this is now available in PM2 2.9.2:

```bash
$ npm install pm2 -g
$ pm2 update
```

**Comment by ledlamp** [comment]
i'm still having this issue with some processes which I have all set the uid for using `--uid` option:

![https://tmov.qonq.gq/](https://tmov.qonq.gq/)

the three processes that show with user "root" in the pm2 list are actually running as users "selfbot" and "owop-bot" as shown in the gnome system monitor.
